plugins/formal-verification/formver.uniqueness/src/org/jetbrains/kotlin/formver/UniqueCheckerContext.kt

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright 2010-2024 JetBrains s.r.o. and Kotlin Programming Language contributors.
+ * Use of this source code is governed by the Apache 2.0 license that can be found in the license/LICENSE.txt file.
+ */
+
+package org.jetbrains.kotlin.formver
+
+import org.jetbrains.kotlin.fir.FirSession
+import org.jetbrains.kotlin.fir.symbols.impl.FirFunctionSymbol
+import org.jetbrains.kotlin.name.ClassId
+
+interface UniqueCheckerContext {
+    val config: PluginConfiguration
+    val errorCollector: ErrorCollector
+    val session: FirSession
+
+    val uniqueId: ClassId
```

Comment:
The role of the classid here isn't quite clear, could you add a comment?

Reply:
Is it really necessary to expose `uniqueId` like this?  Can you provide a function that will do the job?
If it is really necessary, please call this `uniqueAnnotationClassId` and explain why it's necessary in the comment.